core/src/test/scala/org/apache/spark/metrics/MetricsSystemSuite.scala

```diff
@@ -40,7 +40,8 @@ class MetricsSystemSuite extends SparkFunSuite with BeforeAndAfter with PrivateM
   }
 
   test("MetricsSystem with default config") {
-    val metricsSystem = MetricsSystem.createMetricsSystem("default", conf, securityMgr)
+    val metricsSystem =
+      MetricsSystem.createMetricsSystem(MetricsSystemInstances.TEST, conf, securityMgr)
```

Comment:
I think this is meant to be 'default' on purpose?

Reply:
Yes it is on purpose. This test wants to use the "default" name to show the config it used. 
I was trying to add a `MetricsSystemInstances.DEFAULT` for this single test case, but find it hard to understand in instances list. I think it may be acceptable to name it `MetricsSystemInstances.TEST`, just like the test next to it. In fact they have the same config. After all, the test logic differences, not names, are most important.

Reply:
It looks like they test something different though. "default" has no config, but "test" has some config defined in `test_metrics_system.properties`. They assert the difference. 

I don't think we need to define a constant for "default". In fact, we probably don't need a constant for "test"; that can be local to the test class here. It's not something any other code would reference and not a 'real' metrics system.

Reply:
Good idea ! Let me try.

Reply:
updated.